optuna/study/study.py

```diff
@@ -426,7 +426,7 @@ def objective(trial):
             n_trials=n_trials,
             timeout=timeout,
             n_jobs=n_jobs,
-            catch=catch,
+            catch=catch if isinstance(catch, tuple) else (catch,),
```

Comment:
`tuple` should take iterable according to its definition. 

```suggestion
            catch=tuple(catch) if isinstance(catch, Iterable) else (catch,),
```